addons/website_sale/views/templates.xml

```diff
@@ -818,9 +820,7 @@
                                 t-att-value="c.id"
                                 t-att-checked="'true' if c.id == category.id else None"/>
                             <div
-                                t-attf-class="d-flex align-items-center fs-6 fw-normal
-                                    btn btn-{{navClass}} {{'ps-2 pe-3' if c.image_128 else 'px-4'}}
-                                    {{ 'border-primary' if c.id == category.id else '' }}"
+                                t-attf-class="btn btn-{{navClass}} d-flex align-items-center h-100 {{'ps-2 pe-3' if c.image_128 else 'px-4'}}"
```

Comment:
This kind of changes make reading the diff difficult 😉
Could you keep the classes order while tweaking what's necessary only? 
Also, I think that we should follow the "100 characters limit " convention here 😉

Reply:
Splitting a class attribute on multiple lines would render the space in the rendered attribute. We should avoid that, we can either use t-att and split the concatenation on multiple lines or make an exception like I did for this attribute.

Reply:
Since I have to add back the previous condition this would look somewhat like this: 
```xml
<div
    t-attf-class="btn btn-{{navClass
        }} d-flex align-items-center h-100 {{
        'ps-2 pe-3' if c.image_128 else 'px-4'}} {{
        c.id == category.id and 'border-primary'}}"
>
```
Yes it does allow to not render the space in the attribute, but the code is not more readable IMO. If it doesn't provide readability value I don't see the point in not letting it exceed 100char and let it wrap by itself 🤷